edflow/iterators/batches.py

```diff
@@ -84,26 +86,187 @@ def batch_to_canvas(X, cols=None):
     return canvas
 
```

Comment:
maybe document that the other versions are just there to document why v2 was choosen.